sdk/tables/azure-tables/review/tables.api.md

```diff
@@ -443,6 +453,19 @@ export interface TableQueryEntitiesWithPartitionAndRowKeyHeaders {
     xMsContinuationNextRowKey?: string;
 }
 
+// @public
+export type TableQueryEntitiesWithPartitionAndRowKeyResponse = TableQueryEntitiesWithPartitionAndRowKeyHeaders & {
+    [propertyName: string]: any;
```

Comment:
because there's a string indexer here, doesn't that mean we get no strong typing on the result? e.g. you can index into whatever property name you want? 

Would it make sense to make this a generic so if the consumer knows the response shape they can provide an interface for it?

Reply:
Yeah, we shouldn't really be exporting this generated interface. I think this is a side effect on relaying on Omit to override the types. I'll get rid of the Omit types in  #10534  and we should be able to stop exporting these interfaces. And all user-facing ones would have Generic types instead of this indexers.